index.html

```diff
@@ -277,76 +289,94 @@ <h3 class="Skills-title">Skills</h3>
             <li>Terminal</li>
           </ul>
         </div>
-      </section>
-
-			      <!-- Skills end -->
-
-      <!-- About section end -->
-
-      <!-- Form start  -->
-
-      <form action="https://formspree.io/f/xyyvwazk" method="POST">
-        <h2>
-          I'm always intersted in hearing about new projects, so if you'd like
-          to chat please get in touch.
-        </h2>
-        <div class="form">
+      </div>
+    </section>
+
+    <!-- Skills end -->
+
+    <!-- About section end -->
+
+    <!-- Form start  -->
+
+    <form action="https://formspree.io/f/xyyvwazk" method="POST">
+      <h2>
+        I'm always intersted in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <div class="form">
+        <input
+          class="user-name"
+          type="text"
+          name="user_name"
+          placeholder="Full name"
+          maxlength="30"
+          required />
+        <div class="desktop-name">
           <input
+            class="first_name"
             type="text"
-            name="user_name"
-            placeholder="Full name"
+            name="first_name"
+            placeholder="First name"
             maxlength="30"
-            required
-          />
+            required />
```

Comment:
-[ ] There should be one input for Full name (just like the mobile version) and this should take the space of both First Name and Last Name inputs (shown in the desktop version). Kindly remove the inputs for the first and last name and use only the full name input.

Reply:
@leonardodiasb please reply me??????